coralproject/talk
# [next] Apollo Upgrade + Extensions

## What does this PR do?

- Upgraded `apollo-server-express`
- Added a logging extension

## How do I test this PR?

1. Set `LOGGING_LEVEL=debug`
2. Perform a query that causes an error, see the error logged
3. Perform a named query that doesn't cause an error, see the query logged
